NosTests/NoteParserTests.swift

```diff
@@ -4,6 +4,79 @@ import Dependencies
 
 final class NoteNoteParserTests: CoreDataTestCase {
 
+    func testContentWithRawNpubPrecededByAt() throws {
+        // Arrange
+        let npub = "npub1pu3vqm4vzqpxsnhuc684dp2qaq6z69sf65yte4p39spcucv5lzmqswtfch"
+        let hex = "0f22c06eac1002684efcc68f568540e8342d1609d508bcd4312c038e6194f8b6"
+        let content = "You can find me at @\(npub)"
+        
+        let expected = "You can find me at \(npub)"
+
+        // Act
+        let tags: [[String]] = [[]]
+        let context = try XCTUnwrap(testContext)
+        let (attributedContent, _) = NoteParser.parse(
+            content: content,
+            tags: tags,
+            context: context
+        )
+
+        // Assert
+        XCTAssertEqual(String(attributedContent.characters), expected)
+        let links = attributedContent.links
+        XCTAssertEqual(links.count, 1)
+        XCTAssertEqual(links[safe: 0]?.key, npub)
+        XCTAssertEqual(links[safe: 0]?.value, URL(string: "@\(hex)"))
+    }
+
+    func testContentWithRawNIP05() throws {
+        // Arrange
+        let nip05 = "linda@nos.social"
+        let webLink = "https://njump.me/\(nip05)"
+        let content = "hello \(nip05)"
+        let expected = content
+
+        // Act
+        let tags: [[String]] = [[]]
+        let context = try XCTUnwrap(testContext)
+        let (attributedContent, _) = NoteParser.parse(
+            content: content,
+            tags: tags,
+            context: context
+        )
+
+        // Assert
+        XCTAssertEqual(String(attributedContent.characters), expected)
+        let links = attributedContent.links
+        XCTAssertEqual(links.count, 1)
+        XCTAssertEqual(links[safe: 0]?.key, nip05)
+        XCTAssertEqual(links[safe: 0]?.value, URL(string: webLink))
+    }
+
+    func testContentWithRawNIP05AndAtPrepended() throws {
+        // Arrange
+        let nip05 = "linda@nos.social"
+        let webLink = "https://njump.me/\(nip05)"
```

Comment:
We need to define how we want to achieve that, if we want to.

I think we won't want to be doing requests when parsing notes, which is what we need to do in order to know which is the npub. That leave us with two alternatives:

1. Modify ProfileView so that it can be shown with just a NIP-05 (that will first request the npub and then look if we have the profile locally)
2. Don't do anything, just bare NIP-05 identifiers are not good ways of referencing to people in nostr, for example, because it doesn't notify those users you are mentioning them. We should be encouraging users to mention them accordingly in the editor instead of just writing martin@nos.social.

So, as I think any of these alternatives is outside the scope of this ticket, which was just fixing the display issues, I decided to just link to jump as a middle ground alternative that didn't cost me much effort.

Reply:
I see a third way:
1. make all NIP-05s links to a custom URL scheme (nos://nip05/martin@nos.social)
2. Handle this URL scheme in `Router.open(url: URL, with context: NSManagedObjectContext)`
3. Create an observable property like `isLoading` on the `Router`.
4. Use it to show a loading indicator on `AppView` while we look up the NIP-05 (disable UI behind it)
5. If it's a valid NIP-05 and we get an npub, open the profile for that npub. If not, this might be an email address, open the mail app.

This avoids us trying to make web requests when formatting the note contents, at the cost of a little loading time when the user taps. It also solves the problem of email addresses being mistaken for NIP-05s.

Reply:
@mplorentz that would be option 1 in my previous message, we don't need to use the url scheme I we don't want it, we can just open the ProfileView with the same mechanism me open it by following links to @npub1 or MessageView by following links to %note1. I will create a different ticket for this, so we can get these bugfixes I'm adding here in Production until we work on following the NIP-05 identifiers the upcoming sprint.

Reply:
I created an issue here https://github.com/planetary-social/nos/issues/1087 and moved it to this sprint so I work on it in a different branch. I think the njump links, while a bit weird, are functional and we can have this on production until we finish that issue.

Reply:
This makes sense to me and I'm good with it.

Reply:
Please check with Linda on the priority of #1087 if you haven't already.